setup.py

```diff
@@ -54,17 +62,7 @@ def run_tests(self):
     author='Zalando SE',
     url='https://github.com/zalando/lizzy',
     license='Apache License Version 2.0',
-    install_requires=['connexion>=0.13',
-                      'environmental>=1.1',
-                      'decorator',
-                      'pystache',
-                      'pytz',
-                      'pyyaml',
-                      'rod',
-                      'stups-kio',
-                      'stups-senza>=1.0.40',
-                      'uwsgi',
-                      'uwsgi_metrics3'],
+    install_requires=get_install_requirements('requirements.txt'),
```

Comment:
It breaks when you try to install the packaged version since the requirements.txt is not being included in the .egg/.whl file.